header.php

```diff
@@ -31,6 +31,7 @@
 </head>
 
 <body <?php body_class(); ?>>
+<?php do_action( 'wp_body_open' ); ?>
```

Comment:
`do_action( 'wp_body_open' )` is already defined in [/wp-includes/general-template.php](https://github.com/WordPress/WordPress/blob/master/wp-includes/general-template.php#L2765). To trigger the `wp_body_open` action, one can call simply the `wp_body_open()` function.

Reply:
This function simply triggers `do_action( 'wp_body_open' )`. Technically, we can call `do_action( 'wp_body_open' )` directly. It's only a matter of style

Reply:
`wp_head()` and `wp_footer()` are also wrappers for `do_action( 'wp_head' )` and `do_action( 'wp_footer' )`, and yet, using these directly is not recommended.